www/src/models/mineralSite/MineralSite.ts

```diff
@@ -16,10 +16,10 @@ export type FieldEdit =
   | { field: "location"; value: string }
   | { field: "depositType"; observedName: string; normalizedURI: string }
   | {
-      field: "grade";
-      value: number;
-      commodity: string;
-    }
+    field: "grade";
+    value: number;
+    commodity: string;
+  }
```

Comment:
Please setup auto formatting on vscode. I already mentioned this multiple times

Reply:
I have already done that